pkg/models/pgdb/organizationUserTable.go

```diff
@@ -37,6 +37,32 @@ func (s DbPermission) String() string {
 	return "NoPermission"
 }
 
+func (s DbPermission) AsOrganizationRole() string {
+	switch s {
+	case NoPermission:
+		return "none"
+	case Guest:
+		return "guest"
+	case Read:
+
+		return "collaborator"
+	case Write:
+
+		return "collaborator"
+	case Delete:
+
+		return "collaborator"
```

Comment:
I'm not familiar with OrganizationRoles; Read, Write, and Delete all map to collaborator? Is this just a reflection of the fact that a user with collaborator role in an org can look at, create, and delete datasets in that org? Like there is no non-guest role in the org where you can view datasets but not create them? Org permissions are not that fine grained?

Reply:
After a closer look at the `auth-middleware` repo and the Pennsieve `core-models` implementation -- I updated the mapping of `DbPermission` to a string representation of Role, and also the reverse mapping. There is more commentary in this PR's description.

Reply:
Thanks @muftring!